metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ghoneycutt-nrpe",
-  "version": "1.1.1",
+  "version": "1.1.2",
```

Comment:
This comes from your commit after merging @Phil-Friderici 's changes.
It appears in my initial PR probably because i updated from the upstream. Should i undo? But then i will miss the other upstream changes.